internal/install/license_key_fetcher.go

```diff
@@ -34,6 +35,6 @@ func (f *ServiceLicenseKeyFetcher) FetchLicenseKey(ctx context.Context) (string,
 		return "", err
 	}
 
-	log.Debugf("Found license key %s", licenseKey)
+	log.Debugf("Found license key %s", utils.Obfuscate(licenseKey))
```

Comment:
👍

Reply:
@sanderblue for the story, I started implemented an `Obfuscate` method in that same file, and then I thought, this is pretty generic, I should move it to a more global scope, so I search and I found the utils package. That looks like a right place. As I scroll down I found the exact same implementation of the Obfuscate method. I happily deleted my new implementation :)